src/features/AboutProject/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Link } from 'react-router-dom'
+import { PLANET_OBSERVER_PATH } from '../Router/pathes.ts';
+
+
+const AboutProject = () => {
+    return (
+        <ul>
+        <li>
+          <Link to={PLANET_OBSERVER_PATH}>Planet Observer</Link>
+        </li>
+      </ul>  
+    )
+}
+
+export default AboutProject
```

Comment:
You have to apply prettier changes onSave action in VS code. So, here should be semicolon.

Reply:
This changes for whole PR.